src/group_theory/monoid_localization.lean

```diff
@@ -92,7 +92,7 @@ satisfies this predicate, then `N` is isomorphic to the localization of `M` at `
 extends monoid_hom M N :=
 (map_units' : ∀ y : S, is_unit (to_fun y))
 (surj' : ∀ z : N, ∃ x : M × S, z * to_fun x.2 = to_fun x.1)
-(eq_iff_exists' : ∀ x y, to_fun x = to_fun y ↔ ∃ c : S, x * c = y * c)
+(eq_iff_exists' : ∀ x y, to_fun x = to_fun y ↔ ∃ c : S, ↑c * x = c * y)
```

Comment:
The original order of multiplication is compatible with the noncommutative Ore localization of monoids, [see comment](https://github.com/leanprover-community/mathlib/pull/14348#discussion_r914404010), while the new order is not. If this PR is merged, and if we want to make Ore localization a `localization_map`, then we need to switch left and right in the definition of `ore_set` and everywhere in #14348. Is that the way to go?

Reply:
I agree that it might be reasonable to swap the order of multiplication there too; but I decided not to do so in this PR.